### PR TITLE
docs: created a new document for observability

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: docs/main/user-guides/observability.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+- Grafana to visualize your data
+
+Prometheus and Grafana, together provide a robust solution for collecting metrics, visualizing data, and creating alerts.
+
+## Monitoring Setup
+
+OpenEBS provides a basic cloud-native monitoring stack built using Prometheus and Grafana, as an add-on Helm chart. This has pre-configured dashboards for visualization of metrics from the various OpenEBS storages.
+
+:::info
+A set of Grafana dashboards for monitoring and visualization of exported metrics via various OpenEBS storage are bundled with OpenEBS installation which is by default **disabled**. Grafana will be using Prometheus as a data source.
```
Comment (w3aman) [inline docs/main/user-guides/observability.md:26]: this part is not true currently, as it is removed from umbrella chart of openebs installation.

File: docs/main/user-guides/observability.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+ - Monitoring
+ - Observability
+ - Pool metrics
+ - Prometheus
+ - Grafana
+description: This guide explains about monitoring your cluster.
+---
+# Observability
+
+Monitoring your cluster is essential to ensure the health, efficiency, and reliability of your storage infrastructure.​ The following technologies are implemented to monitor your cluster:
```
Comment (abhilashshetty04) [inline docs/main/user-guides/observability.md:14]: No comma required after efficiency?

File: docs/main/user-guides/observability.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+
+- Prometheus to collect data
+- Grafana to visualize your data
+
+Prometheus and Grafana, together provide a robust solution for collecting metrics, visualizing data, and creating alerts.
```
Comment (abhilashshetty04) [inline docs/main/user-guides/observability.md:19]: Here as well, no comma after visualizing data..

File: docs/main/user-guides/observability.md (hunk 2 above)
Comment (balaharish7) [reply docs/main/user-guides/observability.md:14]: @abhilashshetty04 As per the Documentation style guide a "Comma" is required in a series consisting of three or more elements.

File: docs/main/user-guides/observability.md (hunk 3 above)
Comment (balaharish7) [reply docs/main/user-guides/observability.md:19]: @abhilashshetty04  As per the Documentation style guide a "Comma" is required in a series consisting of three or more elements.

**Review by w3aman — APPROVED** [review]

**Review by abhilashshetty04 — APPROVED** [review]

**Review by avishnu — APPROVED** [review]

**Pull request merged**